soc/arm/nordic_nrf/nrf53/power.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ * Copytight (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <power/power.h>
+
+#ifdef CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1
+#include <hal/nrf_regulators.h>
+#endif
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
+
+/* Invoke Low Power/System Off specific Tasks */
+void sys_set_power_state(enum power_states state)
+{
+	switch (state) {
+#ifdef CONFIG_SYS_POWER_DEEP_SLEEP_STATES
+ #ifdef CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1
```

Comment:
I don't believe we generally indent preprocessor directives.

Reply:
I left it like this to keep conistent with `nrf52/power.c`. Should I delete indents here and in `nrf52`?

Reply:
No, let's leave that for some future progress toward #21392.  I'd rather have less noise with internal consistency, than partial steps toward project-wide consistency.  (There are 570 indented `#if*` statements in Zephyr, compared to 15289 that are not.)

Reply:
We should not indent preprocessor directives, that's true